### ISSUE TITLE
weird row count behavior in editor window

### ISSUE THREAD

**Issue by lersek** [unixwork/xnedit#125]
This is with xnedit built at commit b1c382a ("mark iconified windows in the window menu"), against Motif 2.3.8.

In `~/.Xresources`, I have `nedit.textRows: 63`. I observe the following behavior:

* Start `xnc` from a terminal (such that `xnedit` is not yet running). In the window that appears, I have room for 63 lines, as expected.
* Start `xnc a.txt` from the same terminal. In the editor window, the title bar changes to `a.txt`, and I still have room for 63 lines, as expected.
* Start `xnc b.txt` from the same terminal. In the editor window, the tab bar with tabs `a.txt` and `b.txt` appears. The text edit area is still 63 lines tall, which is a problem, because now the window is too tall (it doesn't fit my screen any longer), due to the tab bar taking up vertical space, and the same-size textedit area being shifted down.

Contrast with:

* Start `xnc a.txt b.txt` from a terminal (such that `xnedit` is not yet running). In the window that appears, I have the tab bar, and both text files open, and vertical space for *61* lines. This is the proper behavior: the text area is auto-reduced by two lines, for compensating for the tab bar. This way the editor window still fits my screen.
* Now press `Ctrl-F`, for opening the search dialog -- or even just press `Alt-Tab`, for moving the focus from the editor window to the terminal. At this point, the editor window is resized to 63 rows height, making the editor window too tall again.

So, basically, two avenues to the same bug (incorrect number of rows), even though in the second reproducer, the initial appearance, together with the tab bar, is correct (61 lines).

Using the same motif and xnedit binaries, the above symptoms do *not* show up (and everything is correct), on a different computer. The differences are that
* the problematic setup runs RHEL-9, with icewm 3.4.1-1.el9, and the `nedit.textRows` entry is in `~/.Xresources`,
* while the okay setup runs RHEL-7, with icewm 1.3.8-9.el7, and the `nedit.textRows` entry is in `~/.xnedit/nedit.rc`.

I expect this isn't actually an xnedit or motif bug, but a problem with something "underneath". Where should I look? Thanks.

**Comment by lersek** [comment]
... to clarify, on the good (RHEL-7) computer, the first reproducer reduces the number of lines by two, when `b.txt` is opened, and the second reproducer does not increase the number of lines by two when the focus is moved elsewhere. All in all, the good setup maintains the line count in the textedit area such that the tab bar (if any) plus the line count precisely fill up my screen height, under all circumstances.

**Comment by unixwork** [comment]
I'm not sure, I guess the behavior depends on the window manager. I will try to reproduce this with icewm.
